README.md

```diff
@@ -69,6 +69,66 @@ module "guardduty" {
     }
   }
 }
+
+```
+**Important:** It is correct that the account delegated as GuardDuty Admin (commonly Security) is not added in the `guardduty_member_accounts` list.
+
+### Import Detector Module 
+To import the above module, you need to know the AWS GuardDuty Detector ID was enabled in the delegation process on the selected admin account. You can use aws-cli to quickly get these attributes
+
+- Install [aws-cli]
+
+- #### When you are using only `Terraform` (without a cli/wrapper tool)
+```shell
+## 1: Get AWS GuardDuty Detector ID
+
+aws guardduty list-detectors --profile [AWS_PROFILE]
+
+## 2: You should see a an output like this
+
+{
+    "DetectorIds": [
+        "XXXXXXX"
+    ]
+}
+
+## 3: Replace DETECTOR_ID and run Terraform import
+
+terraform import module.guardduty.aws_guardduty_detector.this [DETECTOR_ID] 
+
+## Optional All-in-one Import Command
+
+terraform import module.guardduty.aws_guardduty_detector.this $(aws guardduty list-detectors --profile [AWS_PROFILE] --query 'DetectorIds[0]') 
+
+```
+
+- #### When you are using `binbash Leverage-cli`
```

Comment:
Since the Leverage docker image already bundles aws-cli this can also be executed as `leverage aws guardduty list-detectors --profile [AWS_PROFILE]` whithout the need of exporting the credentials environment variables. Making the process somewhat simpler!

Reply:
@angelofenoglio Great!

I try these steps and add them as an option.

Thanks for the update!

Reply:
@angelofenoglio Maybe, this addition to the readme will leave it as an issue to be added later and release the version with the functionality changes.
I think you have to leave both options because not all users of the module work from leverage-cli.